scripts/run_simulations.py

```
@@ -0,0 +1,157 @@
+            # Priors
+            if variant_run_config.get("n_prior_included") is None:
+                if dataset_n_prior_included is not None:
+                    variant_run_config["n_prior_included"] = dataset_n_prior_included
+
+            if variant_run_config.get("n_prior_excluded") is None:
+                if dataset_n_prior_excluded is not None:
+                    variant_run_config["n_prior_excluded"] = dataset_n_prior_excluded
+
+            # Ensure 'id' is present for output naming, even if not in original variant config
+            if "id" not in variant_run_config:
+                 variant_run_config["id"] = variant_id
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)